cwl/src/main/scala/cwl/internal/EcmaScriptEncoder.scala

```diff
@@ -106,10 +102,7 @@ class EcmaScriptEncoder(ioFunctionSet: IoFunctionSet) {
       Option("nameroot" -> (File.nameroot(file.value) |> ESPrimitive)),
       Option("nameext" -> (File.nameext(file.value) |> ESPrimitive)),
       file.checksumOption.map("checksum" -> ESPrimitive(_)),
-      if (withSize) 
-        sync(file.withSize(ioFunctionSet)).toOption.flatMap(_.sizeOption).map(Long.box).map("size" -> ESPrimitive(_))
-      else
-        None,
+      file.sizeOption.map(Long.box).map("size" -> ESPrimitive(_)),
       Option("secondaryFiles" -> encodeFileOrDirectories(file.secondaryFiles, withSize = false)),
```

Comment:
`withSize` should be removed, as it does not appear to be used, only recursively-plumbed-then-ignored.